src/components/entity/show.vue

```diff
@@ -166,6 +166,12 @@ const breadcrumbLinks = computed(() => [
 
 </script>
 
+<style lang="scss">
+  .page-section-heading {
+    font-size: 24px;
```

Comment:
Is this style meant to be applied on all pages? If so, I think it should be added to `PageSection`. If it's only for this page, I think it needs to be scoped, e.g., add `id="entity-show"` to the root element, then make the selector `#entity-show .page-section-heading`. Right now, I think if you navigate to this page, then navigate in the same browser tab to another page that renders a `.page-section-heading`, this style will apply on that other page as well.